LightCore.Integration.AspNetCore/LightCoreExtension.cs

```diff
@@ -39,8 +45,19 @@ private static void RegisterServices(IContainerBuilder builder, IServiceCollecti
 
         private static void Register(IContainerBuilder builder, ServiceDescriptor serviceDescriptor, ILifecycle lifecycle)
         {
+
             if (serviceDescriptor.ImplementationType != null)
             {
+                var serviceTypeInfo = serviceDescriptor.ServiceType.GetTypeInfo();
```

Comment:
What is this suppose to do?

Reply:
nothing. just code to remove. I added it to easily get some more information during debugging.
I'l remove it

Reply:
OK!

Reply:
just code to remove